modmesh/app/euler1d.py

```diff
@@ -727,8 +727,9 @@ def log(msg):
 
         :return: None
         """
-        sys.stdout.write(msg)
-        sys.stdout.write('\n')
+        if sys.stdout != None:
```

Comment:
What about adding a comment pointing to https://github.com/solvcon/modmesh/issues/334 ?  We may need some refactoring before being able to make a regression (unit) test.  Some explanation in code will help reduce the chance of regression before that.

Reply:
Added!